tutorials/rise_timeseries_frb.ipynb

```diff
@@ -0,0 +1,545 @@
+{
```

Comment:
<p>Small detail: you consider 2 modifications and then you address them in reverse order ;-) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Good catch! Fixed :)</p><p> </p>